### PR TITLE
Update docs for 0.15 DB Layers

### PR DESCRIPTION
**Pull request by gewenyu99**
This pull requests updates documentation in preparation for 0.15 DB Layers changes.

### CODE REVIEW

File: app/views/docs/database.phtml [hunk 1]
```diff
@@ -1,22 +1,35 @@
 <p>
     The Database Service allows you to store your application and users' data and fetch it using different supported queries.
```
Comment (lohanidamodar) [inline app/views/docs/database.phtml:2]: The `Database` service should now be `Databases` everywhere

File: app/views/docs/database.phtml [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
-    Using your Appwrite Database, you can organize your data into collections and documents using the Appwrite simple REST API. You can also use the Appwrite <a href="/docs/realtime">Realtime API</a> to subscribe to live changes in your collections and documents.
+    Using your Appwrite Database, you can create multiple databases, organize your data into collections and documents using the Appwrite REST API. You can also use the Appwrite <a href="/docs/realtime">Realtime API</a> to subscribe to live changes in your collections and documents.
     In addition, the Database Service provides built-in validation to check the integrity of your incoming data, custom indexing for query performance,
     as well as a flexible permissions mechanism to allow you to easily segment data between different users, teams, and roles.
 </p>
 
+<h2><a href="/docs/database#database" id="database">Create Your Database</a></h2>
```
Comment (lohanidamodar) [inline app/views/docs/database.phtml:8]: `/docs/databases/#database`

* Reminder, Appwrite homepage repo might need updates too with renaming links
* Other places might have this link, where we need update too

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Looking great, few changes.

**Comment by eldadfux** [comment]
We should also update realtime docs for new database event names:
https://appwrite.io/docs/realtime

**Comment by eldadfux** [comment]
Same here: https://appwrite.io/docs/webhooks#events

File: app/views/docs/database.phtml (hunk 2 above)
Comment (gewenyu99) [reply app/views/docs/database.phtml]: fixed

File: app/views/docs/database.phtml (hunk 1 above)
Comment (gewenyu99) [reply app/views/docs/database.phtml]: fixed

**Comment by gewenyu99** [comment]
@lohanidamodar I need a hand on a few things:
- What are the new events we introduced?
- What are the new channels we introduced?
- Is 0.15.x stable to build from source? I want to update the example response in realtime guide
- We should also document new functionality introduced in CLI, is there a version of the CLI that we generated which I can reference?

**Comment by gewenyu99** [comment]
This is related to:
https://github.com/appwrite/homepage/pull/44

**Comment by lohanidamodar** [comment]
> @lohanidamodar I need a hand on a few things:
> 
> * What are the new events we introduced?
> * What are the new channels we introduced?
> * Is 0.15.x stable to build from source? I want to update the example response in realtime guide
> * We should also document new functionality introduced in CLI, is there a version of the CLI that we generated which I can reference?

1. All existing database related events will be prefixed by `databases.databaseId.`, new database related events `databases.*`, `databases.databaseId`, `databases.databaseId.create`, `databases.databaseId.read`, `databases.databaseId.update` and `databases.databaseId.delete`.
2. All database related channels will be prefixed by `databases.databaseId.`
3. 0.15.x should be stable to build from source as far as I remember
4. @christyjacob4  or @Meldiron  should know more about CLI

File: app/views/docs/storage.phtml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 <h2><a href="/docs/storage#bucket" id="bucket">Buckets</a></h2>
 
-<p>With version 0.13 of Appwrite, we also have introduced storage buckets. Storage buckets are similar to collections we have in our <a href="/docs/database">Database</a> service. The difference is buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket, whether or not to encrypt the files, scan with antivirus, and more. Let's look at how we can create a bucket and configure it to your needs.</p>
+<p>With version 0.13 of Appwrite, we also have introduced storage buckets. Storage buckets are similar to collections we have in our <a href="/docs/databases">Database</a> service. The difference is buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket, whether or not to encrypt the files, scan with antivirus, and more. Let's look at how we can create a bucket and configure it to your needs.</p>
```
Comment (lohanidamodar) [inline app/views/docs/storage.phtml:6]:
```suggestion
<p>With version 0.13 of Appwrite, we also have introduced storage buckets. Storage buckets are similar to collections we have in our <a href="/docs/databases">Databases</a> service. The difference is buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket, whether or not to encrypt the files, scan with antivirus, and more. Let's look at how we can create a bucket and configure it to your needs.</p>
```

**Review by lohanidamodar — APPROVED** [review]
LGTM

**Pull request merged**